UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs

```diff
@@ -1,159 +1,44 @@
-﻿using System;
-using HealthV2;
-using Player.Movement;
+﻿using HealthV2;
+using System;
 using UnityEngine;
 
 namespace Items.Implants.Organs
 {
-	public class BodyFat : BodyPartFunctionality, IMovementEffect
+	public class BodyFat : BodyPartFunctionality
 	{
-		[SerializeField] private float maxRunSpeedDebuff = -2;
-		[SerializeField] private float maxWalkingDebuff = -1.5f;
-		[SerializeField] private float maxCrawlDebuff = -0.2f;
+		public float CurrentNutrient { get; private set; } = 0;
```

Comment:
what did you do with the movement debuff ?

Reply:
Its in DigestiveSystem.cs

Reply:
Having multple gameobjects with their own hunger fighting for movement wasnt good. So digestive system handles movement based on the unified hungerstate

Reply:
but Having multiple fat objects allowed obesity to work pretty seamlessly, because if you have more fat objects you have more debuffs